### ISSUE TITLE
[FEATURE] Pa11y Scanning

### ISSUE THREAD

**Issue by BryanJames16** [BryanJames16/RelaySpark#30]
## 🚩 Problem Statement
No support for accessibility scanning jobs.

## 👍 Description
Inclusion of pa11y scanning jobs for accessibility violation checks.

## 🏝️ Alternatives
Use of `axe-scan` for accessibility checking.

## 🏟️ Additional Context
- https://github.com/pa11y/pa11y-ci
